package-res/resources/web/prompting/builders/TextInputBuilder.js

```diff
@@ -15,16 +15,53 @@
  *
  */
 
+/**
+ * <h2>The Text Input Builder</h2>
+ *
+ * To use the TextInputBuilder you should require the appropriate file
+ * from Common-Ui:
+ *
+ * <pre><code>
+ *   require(['common-ui/builders/TextInputBuilder'], 
+ *     function(TextInputBuilder) { 
+ *       
+ *     }
+ *   );
+ * </code></pre>
+ * 
+ * To get the component you'll have to create a new instance of the builder and
+ * call the <code>build</code> method:
+ *
+ * <pre><code>
+ *   var textInputBuilder = new TextInputBuilder();
+ *
+ *   var textInputComponent = textInputBuilder.build(args);
+ * </code></pre>
+ *
+ * where 'args' is an object that contains the prompt panel and the parameters
+ * necessary for the component as per [the CDF documentation]{@link http://localhost:8080/pentaho/api/repos/:public:plugin-samples:pentaho-cdf:pentaho-cdf-require:30-documentation:30-component_reference:10-core:37-TextInputComponent:text_input_component.xcdf/generatedContent}.
```

Comment:
here is another proposal i worked on: https://github.com/pentaho/pentaho-platform-plugin-common-ui/pull/454
I was thinking... and for some cases it could probably makes sense to have an js object with the example of the args. We could end having that, as an example and also point to the CDF documentation in case someone needs to have more information about extra configuration and/or the types of each property.
Please let me know what you guys think